doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -130,6 +129,23 @@ The following rules govern create, read, update, and delete (CRUD) level access.
 <a name="6.3.4"></a>
 ### 6.3.4 Workload Security
 
+OpenStack segregates its infrastructure (for example, hosts) by Regions, Host aggregates and Availability Zones (AZ). Workloads can also be segregated by server groups (affinity and non-affinity groups). These options support the workloads placement requirement _sec.wl.001_.
+
+Separation of non-production and production workloads, or by workload category (for example, payment card information, healthcare, etc.) requires separation through server groups (Regions, AZs) but also requires network and storage segregation as in Regions but also AZs if engineered to do so. Thus, the separation of these workloads is handled through placement of workloads in separate AZs and/or Regions (_sec.wl.005_ and _sec.wl.006_).
```

Comment:
`...separation through server groups ...` -  Server groups can be misinterpreted as OpenStack affinity and anti-affinity filters which is not what Region and AZs mean. 
Needs a re-wording.

Reply:
@deebhatia added "for example". Hope that helps. Thanks

Reply:
Maybe it's only me. Let's go ahead with server groups. If someone else raises it again, we may reword it later on.